be/src/util/jsonb_parser_simd.h

```diff
@@ -296,21 +298,27 @@ class JsonbParserTSIMD {
                 return;
             }
         } else if (num.is_int64() || num.is_uint64()) {
-            if (num.is_uint64() && num.get_uint64() > std::numeric_limits<int64_t>::max()) {
+            if (num.is_uint64() && num.get_uint64() > std::numeric_limits<int128_t>::max()) {
```

Comment:
get_uint64 will return uint64_t is will nerver larger than `std::numeric_limits<int128_t>::max()`

Reply:
deleted